### ISSUE TITLE
compile fails

### ISSUE THREAD

**Issue by gacolitti** [derekwolfson/tex3pt#40]
There seems to be an issue causing compile to fail. I'm working with one of the examples from the help file. This returns the error "(Compile failed - Check pdflatex error or compile manually)":
```
sysuse auto, clear
replace price=price/1000

qui eststo example111: reg price mpg weight
qui eststo example112: reg price mpg weight headroom trunk length
qui eststo example113: reg price mpg weight i.foreign
qui eststo example114: reg price mpg weight headroom trunk length i.foreign

esttab example11* using "test11", ///
replace fragment booktabs ///
label b(3) se(3) se ///
star(* .1 ** .05 *** .01) ///
mgroups("Group 1" "Group 2", ///
pattern(1 0 1 0) ///
prefix(\multicolumn{@span}{c}{) ////
suffix(}) ///
span erepeat(\cmidrule(lr){@span}) ///
) ///end mgroups
stats(r2 N, ///
fmt(3 0) ///
layout("\multicolumn{1}{c}{@}" "\multicolumn{1}{c}{@}") ///
labels("R-Sq" "Observations") ///
) // end stats

tex3pt "test11.tex" using "master11.tex", ///
page ///
clearpage compile ///
title("Tex3pt Example Table") ///
tlabel("tab1") ///
star(ols) ///
note("This is a note in tex3pt!")
```

I can call pdflatex manually with the help of the `whereis` package and it works fine:

```
whereis pdflatex
! "`r(pdflatex)'" master11.tex
```

**Comment by derekwolfson** [comment]
Maybe best to build in the whereis bit to the compile code.  

What OS are you using?

**Comment by gacolitti** [comment]
I'm using macOS Catalina Version 10.15.3.

**Comment by derekwolfson** [comment]
I didn't build the software to compile on Mac yet -- but "whereis" seems like a lovely addition and is pretty scalable.

**Comment by derekwolfson** [comment]
Well -- I think I may have stumbled into why you are having a problem -- I actually just ran into this issue on Overleaf -- it has to do with the migration from Miktek 2020 and TexLive 2021.
[https://tex.stackexchange.com/questions/567985/problems-with-inputtable-tex-hline-after-2020-fall-latex-release](https://urldefense.proofpoint.com/v2/url?u=https-3A__tex.stackexchange.com_questions_567985_problems-2Dwith-2Dinputtable-2Dtex-2Dhline-2Dafter-2D2020-2Dfall-2Dlatex-2Drelease&d=DwMFaQ&c=WO-RGvefibhHBZq3fL85hQ&r=kbuktPDRDiinuCAwk2tfnbnTKvxoHUUTp79EUzsnLEI&m=3GZz36hmlTgW_eRXS2f4v5NBrpe2KgcZLJhHeyerQhN1Vo_Q5preYF5SwiZydwL8&s=JzOza7uOA1Hi-uXDmaDvj6YJo8bgfGMm3CSJU3ZPOZE&e=)

```
\newcommand{\estwide}[3]{
               \vspace{.75ex}{
                       \textsymbols
                       \begin{tabular*}
                       {\textwidth}{@{\hskip\tabcolsep\extracolsep\fill}l*{#2}{#3}}
                       \toprule
                       \estinput{#1} 
                       \bottomrule
                       \addlinespace[0.75ex]
                       \end{tabular*}
                       }
               }       
```

Turns out you cannot do something like this anymore 
```
                       \estinput{#1} 
                       \bottomrule
```

\bottomrule can not be on it's own line anymore it needs to look like ``\\ \bottomrule`` ... which is supremely annoying since this is how esttab outputs things... and there's no real way to edit that code in a meaningful or automated way.

Surprised this hadn't come up sooner -- but if you run on a 2020 TexLive or pre-fall 2020 Miktek -- Tex3pt will work -- otherwise, it'll need some work -- as I'll have to figure out how to call the "old" version on \estinput (based on \input command).

What a freakin' annoying error...
